lib/simplecov/exit_codes/maximum_coverage_drop_check.rb

```diff
@@ -40,23 +42,31 @@ def last_run
 
       def coverage_drop_violations
         @coverage_drop_violations ||=
-          compute_coverage_drop_violations.select do |achieved|
+          compute_coverage_drop_data.select do |achieved|
             achieved.fetch(:max_drop) < achieved.fetch(:drop_percent)
           end
       end
 
-      def compute_coverage_drop_violations
+      def compute_coverage_drop_data
         maximum_coverage_drop.map do |criterion, percent|
           {
             criterion: criterion,
             max_drop: percent,
-            drop_percent: last_run[:result][criterion] -
+            drop_percent: last_coverage(criterion) -
               SimpleCov.round_coverage(
                 result.coverage_statistics.fetch(criterion).percent
               )
           }
         end
       end
+
+      def last_coverage(criterion)
+        last_run[:result][criterion].tap do |last_coverage_percent|
+          if !last_coverage_percent && criterion == "line"
+            return last_run[:result][:covered_percent]
```

Comment:
I don't think this works :thinking: 

```
irb(main):001:0> true.tap do return false end
Traceback (most recent call last):
        6: from /home/tobi/.asdf/installs/ruby/2.7.2/bin/irb:23:in `<main>'
        5: from /home/tobi/.asdf/installs/ruby/2.7.2/bin/irb:23:in `load'
        4: from /home/tobi/.asdf/installs/ruby/2.7.2/lib/ruby/gems/2.7.0/gems/irb-1.2.6/exe/irb:11:in `<top (required)>'
        3: from (irb):1
        2: from (irb):1:in `tap'
        1: from (irb):1:in `block in irb_binding'
```

As in can't return from inside the block and iirc tap just returns self anyhow. I think a "normal" if without tap should do the trick.

Also made me realize CI didn't run.. grmll.